test/CreatorToken.t.sol

```diff
@@ -242,6 +242,123 @@ abstract contract Buying is CreatorTokenTest {
     vm.stopPrank();
   }
 
+  function test_BulkBuy(address _buyer, uint256 _numTokensToBuy) public {
+    _assumeSafeBuyer(_buyer);
+    _numTokensToBuy = bound(_numTokensToBuy, 1, 100);
+
+    uint256 _originalPayTokenBalanceOfCreatorTokenContract =
+      payToken.balanceOf(address(creatorToken));
+    uint256 _originalBuyerBalanceOfCreatorTokens = creatorToken.balanceOf(_buyer);
+    uint256 _originalCreatorTokenSupply = creatorToken.totalSupply();
+    uint256 _originalPayTokenBalanceOfBuyer = payToken.balanceOf(_buyer);
+    uint256 _originalPayTokenBalanceOfCreator = payToken.balanceOf(creator);
+    uint256 _originalPayTokenBalanceOfAdmin = payToken.balanceOf(admin);
+
+    uint256 _expectedPayTokenAddedToContract;
+    uint256 _expectedTotalPricePaidByBuyer;
+    uint256 _expectedPayTokenEarnedByCreator;
+    uint256 _expectedPayTokenEarnedByAdmin;
+
+    uint256 _preMintOffset = referrer == address(0) ? 1 : 2;
+
+    for (uint256 _i = 1; _i <= _numTokensToBuy; _i++) {
```

Comment:
nit: might be good to add a quick comment here that says something like "Calculate the total expected payment by asking the bonding curve and calculating the fees manually"

Reply:
Added the comment here: [2c34a33](https://github.com/showtime-xyz/creator-tokens/pull/29/commits/2c34a337551eba1ec04c5b655d1d87ad7631895e)

Reply:
Yes! I think returning the total price from all buy functions (and net proceeds from sell functions) is helpful for conceivable contract integrations on top of CreatorTokens, so I think it's good to include. LGTM.